corehq/apps/app_manager/tests/test_report_config.py

```diff
@@ -94,6 +94,13 @@ def MAKE_REPORT_CONFIG(domain, report_id, columns=None):
             display="color",
             field="color_94ec39e6"
         ).to_json(),
+        FieldColumn(
+            type='field',
+            aggregation="sum",
+            column_id="count",
+            display="count",
+            field="count"
+        ).to_json(),
```

Comment:
Is the commit message here right? Did this fix a test or did this add a test case that previously would have failed?

Reply:
May be both? The intention was to first fix the failing tests, and it was failing because there was no column "count" present but it was mentioned in the chart configuration. So the report configuration was "incomplete" even if it was not failing earlier.